client-core/src/service/key_service.rs

```diff
@@ -1,9 +1,31 @@
 use secstr::SecUtf8;
 use zeroize::Zeroize;
 
-use client_common::{PrivateKey, PublicKey, Result, SecureStorage, Storage};
+use bip39::{Language, Mnemonic, MnemonicType, Seed};
 
+use client_common::{PrivateKey, PublicKey, Result, SecureStorage, Storage};
 const KEYSPACE: &str = "core_key";
+const KEYSPACE_HD: &str = "hd_key";
+use chain_core::init::network::get_bip44_coin_type;
+use log::debug;
+use tiny_hderive::bip32::ExtendedPrivKey;
+/// Wallet kinds
+/// Basic: default wallet
+/// HD: HD wallet
+/// Hardware: hardware based wallets
+#[derive(Debug, Copy, Clone, Eq, PartialEq)]
+pub enum WalletKinds {
+    /// Basic Wallet
+    Basic,
+    /// HD Wallet
+    HD,
+}
+
+impl Default for WalletKinds {
+    fn default() -> Self {
+        WalletKinds::Basic
+    }
+}
```

Comment:
Can you move this enum to `client_core::types`?

Reply:
ok